src/states/vehicle_state.jl

```diff
@@ -23,7 +23,13 @@ VehicleState() = VehicleState(VecSE2(), NULL_FRENET, NaN)
 VehicleState(posG::VecSE2{Float64}, v::Float64) = VehicleState(posG, NULL_FRENET, v)
 VehicleState(posG::VecSE2{Float64}, roadway::Roadway, v::Float64) = VehicleState(posG, Frenet(posG, roadway), v)
 VehicleState(posG::VecSE2{Float64}, lane::Lane, roadway::Roadway, v::Float64) = VehicleState(posG, Frenet(posG, lane, roadway), v)
-VehicleState(posF::Frenet, roadway::Roadway, v::Float64) = VehicleState(get_posG(posF, roadway), posF, v)
+VehicleState(posF::Frenet, roadway::Roadway, v::Float64) = VehicleState(posg(posF, roadway), posF, v)
+
+posf(veh::VehicleState) = veh.posF
+posg(veh::VehicleState) = veh.posG
+vel(veh::VehicleState) = veh.v
+velf(veh::VehicleState) = (s=veh.v*cos(veh.posF.ϕ), t=s=veh.v*sin(veh.posF.ϕ))
+velg(veh::VehicleState) = (s=veh.v*cos(veh.posG.θ), t=s=veh.v*sin(veh.posG.θ))
```

Comment:
I'm confused by the `t=s=veh.v...` syntax - is this a copy paste error?

Reply:
copy paste error >< should be fixed now